plugins/inventory/stackpath_compute.py

```diff
@@ -0,0 +1,242 @@
+# Copyright (c) 2017 Ansible Project
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+import json
+
+from ansible.errors import AnsibleError
+from ansible.module_utils._text import to_native, to_text
+from ansible.plugins.inventory import (
+    BaseInventoryPlugin,
+    Constructable,
+    Cacheable
+)
+from ansible.utils.display import Display
+
+try:
+    import requests
+except ImportError:
+    raise AnsibleError(
+        'The stackpath_compute dynamic inventory plugin requires requests.'
+    )
+
+DOCUMENTATION = '''
+    name: stackpath_compute
+    plugin_type: inventory
+    short_description: StackPath Edge Computing inventory source
+    requirements:
+        - requests
```

Comment:
```suggestion
        - requests
```
`When fetching URLs, use fetch_url or open_url from ansible.module_utils.urls. Do not use urllib2, which does not natively verify TLS certificates and so is insecure for https.` . it's from https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_best_practices.html#module-security.
can't we use the suggested functions here? (i'm not an expert)